e2e/settings.js

```diff
@@ -24,7 +24,8 @@ module.exports = {
     "G": { "type": "scroll.bottom" },
     "$": { "type": "scroll.end" },
     "d": { "type": "tabs.close" },
-    "D": { "type": "tabs.close.right" },
+    "D": { "type": "tabs.close", "selectLeft": true },
```

Comment:
Only select left?  
Actually I didn't this feature on Vimperator and don't know that importance.  Do users also require "select _right_ after closing tab"?
I think `"select": "left"` option is more flexible.

Reply:
Select right is the default. I originally implemented this as a string (i.e. `{ "select": "left" }` or `{ "select": "right" }`) but switched to boolean since other similar options are booleans, e.g.:

- `{ type: "followStart", "newTab": true }` rather than e.g. `{ "tab": "new" }`
- `{ "type": "command.show.open", "alter": true }` rather than e.g. `{ "mode": "alter" }`

Reply:
Done